js/constants/config.js

```diff
@@ -7,7 +7,7 @@
 // https://vault-staging.brave.com for a dev build
 // http://localhost:3000 for production
 const vaultHost = process.env.VAULT_HOST || 'https://vault-staging.brave.com'
-const adHost = process.env.AD_HOST || '//cdn.brave.com'
+const adHost = process.env.AD_HOST || '//oip.brave.com'
```

Comment:
HTTPS = a non-trusted third party sees your data
HTTP = anyone on your local network, your ISP, the NSA, etc sees your data
so there is still a security gain from HTTPS.

see arguments at https://istlsfastyet.com/ against cost/performance overhead of HTTPS.